fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1039,6 +1041,32 @@ public void testIngestOnSubtree() throws Exception {
 
     }
 
+    /**
+     * Ensure that the objects cannot be pairtree child resources
+     *
+     */
+    @Test
+    public void testIngestOnPairtree() throws Exception {
+        HttpPost method = postObjMethod("");
+        HttpResponse response = client.execute(method);
+
+        //  Following the approach undertaken for FedoraExportIT#shouldRoundTripOnePairtree
+        final String objName = response.getFirstHeader("Location").getValue();
+        final String pairtreeName = objName.substring(serverAddress.length(), objName.lastIndexOf('/'));
```

Comment:
@ajs6f I'm uncertain as to whether this is the case, but I am currently struggling to find this.  Please see the following Gist for a listing of all quads found in respect to the resource "within" the pair-tree "container": https://gist.github.com/jrgriffiniii/598643e2c96379cfaf15